CMake/FindITKUtil.cmake

```diff
@@ -55,7 +55,7 @@ macro(FindITKUtil ITK_VAR_PREFIX )
     ${ARGN}
     )
   list(REMOVE_DUPLICATES ITK_INCLUSION_MODULES)
-  find_package(ITK 4.5 COMPONENTS ${ITK_INCLUSION_MODULES} REQUIRED)
+  find_package(ITK COMPONENTS ${ITK_INCLUSION_MODULES} REQUIRED)
```

Comment:
I have reverted the commit. I thought to be unnecessary since the Superbuild builds a different version of ITK.